src/py/flwr/common/record/configsrecord.py

```diff
@@ -85,3 +85,34 @@ def __init__(
                 self[k] = configs_dict[k]
                 if not keep_input:
                     del configs_dict[k]
+
+    def count_bytes(self) -> int:
+        """Return number of Bytes stored in this object.
+
+        This function counts booleans as ocupying 1 Byte.
+        """
+
+        def get_var_bytes(value: ConfigsScalar) -> int:
+            """Return Bytes of value passed."""
+            if isinstance(value, (int, float)):
+                var_bytes = 4
```

Comment:
do we need to use 64bits for an int?

Reply:
Not necessary for most of time, I think. But it would be a little bit difficult for us to support all int32, int64, float32 and float64. For now, as defined in .proto files, we only use int64 and float64.